content/projects/linux/_index.md

```diff
@@ -61,23 +65,42 @@ Create an empty file named exercise.txt and move this file to the /tmp direc
 Note: foreach instruction write a script and save it under this task
 
 ## Task 6 Find commands
+1. Write a script to find all files ending with 'pdf' on your computer
+2. Write a second command that takes the result of the previous search and copy  into a folder of your choice. 
+3. Write a command to display files that where modified today.
 
-1. Create a script that searches the home directory for all existing files
-2. change directory to My document, create a file named search_result.text. Save and change directory to Descktop
-3. write a scritpt to retrive all directories,files and subdirectories, write the result to search_result.txt
+### Resources 
 
-## Resources
+1. https://www.geeksforgeeks.org/find-command-in-linux-with-examples/
+2. https://unix.stackexchange.com/questions/70455/how-to-run-find-exec-script
 
-1. https://bit.ly/2VQbkDK
 
 ## Task 7 Installing softwares
 
-## Task 8 Linux documentation how to
+
+1. [Presentation](https://docs.google.com/presentation/d/1nDQW1tgDa8CiV2PDQy8pIVh_UliFqMJjOuPbFHFa8Uc/edit?usp=sharing) 
```

Comment:
These links are "edit" links rather than "view" links. Are you giving all the recruits edit access to the presentations? 

Also, what do you think of having a specific folder where we store all publicly readable presentations and documents? Take a look here: https://drive.google.com/drive/u/2/folders/1Sds4qUIi7eFvbj7g82uq6g98QDBE68S9 

Can you please put your presentations there?  And just make sure they have nice names. This presentation has the file name "Field Trip" which is quite misleading.

Reply:
Also, it would be good to put the presentations in the "workshops" directory. If anyone needs to find these presentations then they wont look here. 

Inconsistency == technical debt.  

Let's try keep things consistent.  If you put things in non-obvious places then people are going to waste time looking for things. 

Take a look at this: https://master.d3hv5c7o5j7vym.amplifyapp.com/workshops/how-to-be-a-professional/ 

Please embed your slides in the same way. Or in a better way

Reply:
I have moved all the presentations to the workshop folder. They will show up under the Linux section. 
I have also made them publicly available by moving them in the tech public resources folder.

Thank you for the slide on professionalism. I m with you on that let keep consistency and professionalism in our work. I will definitely work on that.

Thanks again